### ISSUE TITLE
DRAWIO doesnt work with functions!!!

### ISSUE THREAD

**Issue by Arhimagistr** [hediet/vscode-drawio#401]
As i think it's possible to link just classes but not functions and so on. Anybody if you know let know me the tool which helps to debug JS code.. a lot of js code!!

..or may be some ideas how to setup drawio bind not only classes but almost functions etc. ????
P.S.: i've found the vs code plugin: dependency graph... but it works with just dependencies!!! anybody knows how to make functions dependecies map for debugging????

**Comment by abulka** [comment]
As a Python programmer, I agree. The ability to link to functions and files would be greatly appreciated, as Python is not as "class centric" as other programming languages like Java. 

Actually I suppose Javascript programmers would also appreciate being able to link to functions as classes are even rarer in Javascript.

P.S. Being able to link to a `file.function.subfunction` etc. would be pretty cool, as would the ability to link to `file.class.method` or `file.class.attribute` etc.

**Comment by oneezy** [comment]
yeah totally! this would be such a killer feature. it would be extremely useful also if it would just autogenerate the entire folder>file>{function|class|variable|whatever} also. 

i have found some awesome tools for visualizing dependency graphs too but im wanting to go a step further for files. i have found a few tools that kinda fit the bill ill list here:

- https://github.com/pahen/madge
- https://npmgraph.js.org/?q=svelte
- https://npm.anvaka.com/#/view/2d/svelte
- https://www.mermaidchart.com/app/projects/7c185f84-ec8c-4709-9ea4-fac198b6c3cb/diagrams/c0203fc8-cd6b-4397-89cc-c7a48a7c1180/version/v0.1/edit
